### PR TITLE
ATS-125 : Update transform-service-config.json

### PR DESCRIPTION
**Pull request by CezarLeahu**
- remove transformations that are not supported by ATS

### CODE REVIEW

File: src/main/resources/alfresco/transform-service-config.json [hunk 1]
```diff
@@ -27,14 +25,7 @@
       {"value": {"required": false, "name": "targetEncoding"}}
     ],
     "supportedSourceAndTargetList": [
-      {"sourceExt": "pdf",  "maxSourceSizeBytes": 26214400, "targetExt": "txt" },
```
[the remark this answers is not in the document]
Comment (CezarLeahu) [reply src/main/resources/alfresco/transform-service-config.json]: Which paths do we need to support, or which paths do we support at the moment?

We should support, along side **pdf**, all the *office formats* to **txt**.
But we've mistakenly removed all the routes for office formats to text from the T-Router, so right now ATS doesn't support them.
We need to add  those routes back into the T-Router. I'll open an issue for that.

Now, regarding this particular PR, we can either merge this changes now, and later create another PR to add the TIKA options back, or we delay merging this change for a few days, until we fix the routes in the T-Router.

[the remark this answers is not in the document]
Comment (derekhulley) [reply src/main/resources/alfresco/transform-service-config.json]: It'd probably be best to work with stuff that is known and verified to work BUT we are a few days from RC1, still.  You decide on the risk.

[the remark this answers is not in the document]
Comment (CezarLeahu) [reply src/main/resources/alfresco/transform-service-config.json]: Done. I've restored the TIKA routes in the T-Router as part of **ATS-189**. So right now there's no longer a need to remove them from the *transform-service-config.json*.
We can continue with the code review.

[the remark this answers is not in the document]
Comment (montgolfiere) [reply src/main/resources/alfresco/transform-service-config.json]: +1

**Comment by alandavis** [comment]
I will take your word for these transforms not being supported by ATS.
The actual change is fine.

**Comment by alandavis** [comment]
Merged for you so we can get an ACS build together that includes it quickly.

**Pull request merged**